scripts/jenkins/ardana/ansible/post-deploy-checks.yml

```diff
@@ -0,0 +1,30 @@
+- name: Take snapshot of modified/deleted rpm-owned files post deployment
+  shell: rpm -Va | grep -vxFf "{{ pre_deploy_rpm_verification }}"
+  failed_when: false
+  register: rpm_verify_delta
+
+- name: Obtain list of rpm-owned files removed by deployment
+  shell: echo "{{ rpm_verify_delta.stdout }}" | grep '^missing '
+  failed_when: false
+  register: rpm_verify_removed_files
+
+- name: Obtain list of rpm-owned non-config files modified by deployment
+  shell: echo "{{ rpm_verify_delta.stdout }}" | egrep -v '^(missing |.........  c)'
```

Comment:
You might want to ignore ghost files as well. They should show up with a 'g'  in the rpm -Va output. (It might also be an option to call rpm with `--noghost`)

Reply:
Good catch, thanks!!

Reply:
@rhafer Done, although I'm still blocked from testing due to [bsc#1081005 (Ardana horizon venv fails during the installation)](https://bugzilla.suse.com/show_bug.cgi?id=1081005)